conda/recipes/rmm/meta.yaml

```diff
@@ -34,9 +34,11 @@ build:
   ignore_run_exports_from:
     {% if cuda_major == "11" %}
     - {{ compiler('cuda11') }}
+    - cuda-python
```

Comment:
Move this out of the `if`. It's unconditional.

Reply:
Good suggestion, just did that in https://github.com/rapidsai/rmm/pull/1720/commits/84765d347813b0296ed66daf81cf33ad1639d46a